src/plugins/intel_gpu/src/kernel_selector/cl_kernels/sdpa_micro.cl

```diff
@@ -210,6 +210,9 @@ KERNEL(micro_sdpa)(OPTIONAL_SHAPE_INFO_ARG
     Q += (QRY_OFF(b1, b0, 0, 0) + INPUT0_OFFSET);
     V += (VAL_OFF(b1, b0_kv, 0, 0) + INPUT2_OFFSET) / VAL_ELEMENTS_PER_BYTE;
     A += DST_OFF(b1, b0, 0, 0, 0);
+#if WITH_ATTN_MASK
```

Comment:
[random spot] Please add a test case for that

Reply:
@vladimir-paramuzov , there are already test cases covering this scenario :(
They just don't highlight existing issues because of non-representative input data range. With updated input data, this issue can also be seen, as well as others: https://jira.devtools.intel.com/browse/CVS-161873